examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -221,13 +345,17 @@ def create_ad_group(client, customer_id, campaign_resource_name):
     return ad_group_resource_name
 
 
-def create_ad_group_ad(client, customer_id, ad_group_resource_name):
+def create_ad_group_ad(
+    client, customer_id, ad_group_resource_name, customizer_attribute_name
+):
     """Creates ad group ad.
 
     Args:
       client: an initialized GoogleAdsClient instance.
       customer_id: a client customer ID.
       ad_group_resource_name: an ad group resource name.
+      customizer_attribute_name: (optional) the resource name of the customizer attribute
```

Comment:
If it is optional, why don't we supply a default value in the function signature?

Reply:
It's optional because if its present it will create the customizer attribute and ignore if not. I added a note to indicate that.